models/edr/test_results/model_run_results.sql

```diff
@@ -0,0 +1,24 @@
+with dbt_run_results as (
+    select * from {{ ref('dbt_run_results') }}
```

Comment:
One concern about this:
What will happen if the customer disable this model (which now he can, and we recommend him do to so if he wants to disable the artifacts)?